smac/configspace/util.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+import numpy as np
+
+from smac.configspace import Configuration
+
+
+def impute_inactive_hyperparameters(configs: List[Configuration]) -> np.ndarray:
```

Comment:
the name of the method is a bit misleading, since it does more than imputation; it also converts it into our internal numpy representation.